tests/test_cleanup.py

```diff
@@ -92,10 +95,7 @@ class _HandlerWithNullExtractor(Handler):
     """
 
     def calculate_chunk(self, file: io.BufferedIOBase, start_offset: int) -> ValidChunk:
-        pos = file.tell()
-        length = file.seek(0, os.SEEK_END)
-        file.seek(pos, os.SEEK_SET)
-        return ValidChunk(start_offset=0, end_offset=length)
+        return ValidChunk(start_offset=start_offset, end_offset=start_offset + 1)
```

Comment:
Same, as above

Reply:
Actually it is a required change for the tests to continue to work, as the old version would result in a whole file chunk, that is exactly the scope of this PR.